oneview_redfish_toolkit/blueprints/metadata.py

```diff
@@ -39,8 +39,8 @@ def get_metadata():
     """
 
     try:
-        schemas = collections.OrderedDict(util.config.items('schemas'))
-        mtdt = Metadata(schemas)
+        schemas_dict = collections.OrderedDict(schemas.SCHEMAS)
```

Comment:
why do we using a OrderedDict here? Could we use a normal dict?

Reply:
I did'nt want to change the logic of application on this refatoring so I keep it as OrderedDict. But I'm not sure if we need this.

Reply:
😄 👍